<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btn_ts_a">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>741</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>28</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(255, 203, 81, 237), stop:0.886364 rgba(255, 134, 73, 255));
border: 2px solid #ff7474;
border-radius: 10px;
font: 75 28pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Тест level A1-А2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ts_b">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>741</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>28</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(255, 203, 81, 237), stop:0.886364 rgba(255, 134, 73, 255));
border: 2px solid #ff7474;
border-radius: 10px;
font: 75 28pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Тест level B1-B2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ts_c">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>741</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>28</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(255, 203, 81, 237), stop:0.886364 rgba(255, 134, 73, 255));
border: 2px solid #ff7474;
border-radius: 10px;
font: 75 28pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Тест level C1-C2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ts">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>741</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>28</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(255, 203, 81, 237), stop:0.886364 rgba(255, 134, 73, 255));
border: 2px solid #ff7474;
border-radius: 10px;
font: 75 28pt &quot;Comic Sans MS&quot;;
</string>
   </property>
   <property name="text">
    <string>Узнать средний уровень </string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-300</x>
     <y>-200</y>
     <width>1531</width>
     <height>881</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ex">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>520</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>25</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(255, 203, 81, 237), stop:0.886364 rgba(255, 134, 73, 255));
border: 2px solid #ff7474;
border-radius: 10px;
font: 75 25pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;-- Назад</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-10</y>
     <width>901</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(149, 255, 247, 237), stop:0.886364 rgba(201, 107, 255, 255));</string>
   </property>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
  <zorder>btn_ts_a</zorder>
  <zorder>btn_ts_b</zorder>
  <zorder>btn_ts_c</zorder>
  <zorder>btn_ts</zorder>
  <zorder>btn_ex</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
